lib/Wikia/src/Service/User/Attributes/AttributeKeyValueService.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Wikia\Service\User\Attributes;
+
+use Wikia\Logger\Loggable;
+use Wikia\Persistence\User\Attributes\AttributePersistence;
+use Wikia\Util\WikiaProfiler;
+use Wikia\Service\User\Attributes;
+
+class AttributeKeyValueService implements AttributeService {
+
+	const PROFILE_EVENT = \Transaction::EVENT_USER_ATTRIBUTES;
+
+	use WikiaProfiler;
+	use Loggable;
+
+	/** @var AttributePersistence */
+	private $persistenceAdapter;
+
+	/**
+	 * @Inject({
+	 *    Wikia\Persistence\User\Attributes\AttributePersistence::class
+	 * })
+	 * @param AttributePersistence $persistenceAdapter
+	 */
+	function __construct( AttributePersistence $persistenceAdapter ) {
+		$this->persistenceAdapter = $persistenceAdapter;
+	}
+
+	public function setAttribute( $userId, $attribute ) {
```

Comment:
While we're in ```AttributeKeyValueService``` class what do you think about making method names more simple, eg. ```set```, ```get```?